CHANGELOG.md

```diff
@@ -2,14 +2,19 @@
 
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+The format is based on [Keep a Changelog](
+
+),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
 ### Added
 
 - A changelog file.
+- The `asb` resumes unfinished swaps upon startup.
+  This is a breaking changed because we now save the peer-id of the counterparty in the database.
+  It is recommended to reset the database before running the upgraded version of the `asb` because any unfinished swaps will be tried for resume but will fail without the counterparty peer-id being present.
```

Comment:
Also, can we please squash this commit into the one with the actual feature?

Reply:
> This seems to be an unnecessary burden for the user? I'd say let's just skip the swaps where we don't have a PeerId.

Can do - and print a warning in the logs. Otherwise the user does not know that "unsupported" swaps are being skipped.